pkg/trace/config/config_test.go

```diff
@@ -33,6 +33,7 @@ func TestConfigHostname(t *testing.T) {
 	t.Run("fail", func(t *testing.T) {
 		defer cleanConfig()()
 		config.Datadog.Set("apm_config.dd_agent_bin", "/not/exist")
+		config.Datadog.Set("cmd_port", "-1")
```

Comment:
added this to make the test pass on my system (I have dd agent running in the background). 
I think its a worthy addition, to make sure the test will not actually try to connect to the agent, If we only want to test the fallback function